README.md

```diff
@@ -434,8 +434,8 @@ up correctly).
 
 ## Security Notes
 
-Krun is not designed to be run on a production machine, as it uses sudo to
-elevate proveleges to root.
+Krun is not intended to be run on a secure multi-user system, as it uses sudo
+to elevate proveleges.
```

Comment:
You need to check your provelege spelling. [Sorry, terrible joke. I couldn't resist.]